applications/GettingStartedWith/HtmlEditor/index.html

```diff
@@ -0,0 +1,53 @@
+<div id="html-editor">
+    <h2>
+        <img src="/Content/images/doc/21_1/htmlEditor/HtmlEditor.png" alt="HtmlEditor">
+        Formatted Text Editor (HTML Editor)
+    </h2>
+    <br>
+    <p>DevExtreme JavaScript HTML Editor is a client-side WYSIWYG text editor that allows its users to format textual and visual content and store it as HTML or Markdown.</p>
+    <p>Supported features:</p>
+    <ul>
+        <li>Inline formats:
+            <ul>
+                <li><strong>Bold</strong>, <em>italic</em>, <s>strikethrough</s> text formatting</li>
+                <li>Font, size, color changes (HTML only)</li>
+            </ul>
+        </li>
+        <li>Block formats:
+            <ul>
+                <li>Headers</li>
+                <li>Text alignment</li>
+                <li>Lists (ordered and unordered)</li>
+                <li>Code blocks</li>
+                <li>Quotes</li>
+            </ul>
+        </li>
+        <li>Custom formats</li>
+        <li>HTML and Markdown support</li>
+        <li>Mail-merge placeholders (for example, %username%)</li>
+        <li>Adaptive toolbar for working images, links, and color formats</li>
+        <li>Insert images as a link or base64 (drag and drop images to convert them to base64)</li>
```

Comment:
I feel like "base64" acts as an adjective and requires a noun. If it's a noun then it refers to format, not a piece of data. 

```suggestion
        <li>Insert images as a link or a base64 string (drag and drop images to convert them to base64)</li>
```

or 

```suggestion
        <li>Insert images as a link or as base64-encoded binary data (drag and drop images to convert them to base64)</li>
```